LibraBFT/Prelude.agda

```diff
@@ -603,3 +603,13 @@ module LibraBFT.Prelude where
   []       !?      _   = nothing
   (x ∷ _ ) !?      0   = just x
   (_ ∷ xs) !? (suc n)  = xs !? n
+
+  fromToList : ℕ → ℕ → List ℕ
```

Comment:
I think the intended semantics of this should be documented, perhaps including some  examples like:
```
  _ : fromToList 1 1 ≡ 1 ∷ []
  _ = refl

  _ : fromToList 1 2 ≡ 1 ∷ 2 ∷ []
  _ = refl

  _ : fromToList 2 1 ≡ []
  _ = refl
```

Reply:
I added those tests and I added
`-- Like a Haskell list-comprehension for ℕ : [ n | n <- [from .. to] ]`